JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -453,7 +453,7 @@ module Microsoft.ApplicationInsights {
 
         public static parseUrl(url): HTMLAnchorElement {
             if (!UrlHelper.htmlAnchorElement) {
-                UrlHelper.htmlAnchorElement = UrlHelper.document.createElement('a');
+                UrlHelper.htmlAnchorElement = !!UrlHelper.document.createElement ? UrlHelper.document.createElement('a'): {};
```

Comment:
This will result in a url being spat out since it's inspecting the .src property, which in a browser would be parsed and transform.  In a non-browser environment this will just be a no-op and I believe this url should already be a fully-qualified url